frontend/test/metabase/scenarios/filters/relative-datetime.cy.spec.js

```diff
@@ -318,9 +317,9 @@ const setRelativeDatetimeUnit = unit => {
 };
 
 const setRelativeDatetimeValue = value => {
-  cy.findAllByTestId("relative-datetime-value")
+  cy.findByTestId("relative-datetime-value")
```

Comment:
Not sure why you used `findAllBy...` if there's only one field that we're targeting? Even if there were more available, any subsequent action such as `click` or `clear` would've failed because they can be applied to a single element only.